caravel/config.py

```diff
@@ -22,6 +24,9 @@
 # ---------------------------------------------------------
 # Caravel specific config
 # ---------------------------------------------------------
+with open(os.path.join(os.path.dirname(__file__), 'static/assets/package.json')) as package_file:
+    VERSION_STRING= json.load(package_file)['version']
+
```

Comment:
This part is called when config is loaded in __init__.py, I have a feeling it's only called when restarting the server, so VERSION_STRING is not changed, the html templates keep referencing the old version